### PR TITLE
make tests set runtimepath relative to sfile so it works without vundle

### PR DESCRIPTION
**Pull request by edrex**
lifted from https://github.com/junegunn/vader.vim/blob/master/test/vimrc

### CODE REVIEW

**Comment by edrex** [comment]
note this blows away the path rather than appending but that seems ok given that we are trying to make repeatable tests and probably don't want the system plugins.

File: test/init.vim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 set nocompatible
-let &runtimepath = '~/.vim/bundle/wiki.vim,' . &runtimepath
+let &runtimepath = expand('<sfile>:p:h:h') . ',' . &runtimepath
```
Comment (lervag) [inline test/init.vim]: Even simpler:

```vim
let &runtimepath = '../,' . &runtimepath
```

I.e., no reason not to just use relative paths here, is it?

**Comment by lervag** [comment]
Note: This had _nothing_ to do with Vundle or vim-plug, per se, only the fact that I've kept my plugins in `~/.vim/bundle`. I did that already when I started with pathogen many years ago. Now I use vim-plug and the same directory structure, but there is no difference.

Still, I agree that the hardcoded path is an anti pattern.

File: test/init.vim (hunk 1 above)
Comment (edrex) [reply test/init.vim]: sure, looks good

**Comment by edrex** [comment]
Fixed in 0ac5325

**Pull request closed without merging**